src/context/TransactionsProvider/TransactionsProvider.tsx

```diff
@@ -86,11 +85,14 @@ export const TransactionsProvider = ({ children }: TransactionsProviderProps): J
           })
           .map(async chain => {
             const adapter = chainAdapterManager.byChain(chain)
+            // this looks funky, but we need a non zero length array to map over
+            // where we consume it - it either looks weird here or in the consumption
+            // so...  ¯\_(ツ)_/¯
```

Comment:
i understand you moved my comment over but it doesn't make complete sense in this context now (it was written at the implementation side over the consumption side)

Reply:
Yeah probably looks odd now, kept the first line only